adafruit_button.py

```diff
@@ -38,21 +38,50 @@
 
 """
 
+from micropython import const
 import displayio
 from adafruit_display_text.text_area import TextArea
```

Comment:
adafruit_display_text should refer to Label, instead of TextArea (https://github.com/adafruit/Adafruit_CircuitPython_Display_Text/releases/tag/2.0.0)